DOS/migrations/stores.sql

```diff
@@ -5829,3 +5829,392 @@ BEGIN
 
 END;
 $$ LANGUAGE plpgsql;
+
+
+CREATE TYPE grid_renglon_pago AS (
+    serie_folio character varying,
+    saldado boolean,
+    cantidad double precision,
+    tipo_cambio double precision
+);
+
+CREATE FUNCTION public.pago_register(
+    _usuario_id integer,                            --str_data[3]
+    _cliente_id integer,                            --str_data[4]
+    _moneda integer,                                --str_data[7]
+    _fecha_hora timestamp with time zone,           --str_data[8]
+    _banco integer,                                 --str_data[9]
+    _observaciones text,                            --str_data[10]
+    _forma_pago character varying,                  --str_data[11]
+    _cheque character varying,                      --str_data[12]
+    _referencia character varying,                  --str_data[13]
+    _tarjeta character varying,                     --str_data[14]
+    _monto_pago double precision,                   --str_data[16]
+    _fecha_deposito timestamp with time zone,       --str_data[17]
+    _ficha_movimiento_deposito character varying,   --str_data[18]
+    _ficha_cuenta_deposito integer,                 --str_data[19]
+    _ficha_banco_kemikal integer,                   --str_data[20]
+    _tipo_cambio double precision,                  --str_data[21]
+    _anticipo_gastado double precision,             --str_data[22]
+    _no_transaccion_anticipo bigint,                --str_data[23]
+    _saldo_a_favor double precision,                --str_data[24]
+    _grid_detalle grid_renglon_pago[]
+
+) RETURNS character varying LANGUAGE plpgsql AS $$
+
+DECLARE
+
+    -- >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+    -- >> Name:     Registro de pagos                                              >>
+    -- >> Version:  MAZINGER                                                       >>
```

Comment:
Mazinger has gone
Now, we are in galaxy ranger

Reply:
Lo cambio a vanilla-galaxy-ranger